### PR TITLE
feat(tracing-without-performance): Returned orphan errors with trace …

### PR DESCRIPTION
**Pull request by Abdkhan14**
For the ticket: https://getsentry.atlassian.net/browse/PERF-2052.
- We want to add new rows to trace views for traces that contain orphan errors. Currently the trace endpoint ignores orphan errors. 
tracing_without_perf (1)

- This PR aims to append orphan errors to the response of the trace endpoint and should work for both scenarios:
- - Only errors in trace (Scenario 3 from image above).
- - Mixup of errors and transactions (Scenario 2 from image above). 
- Returning `event_type:"error"` and `generation:0` with all errors, to help us identify the orphan errors in the frontend and always group them under the first generation. 
- Added tests for both scenarios.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_trace.py [hunk 1]
```diff
@@ -608,6 +623,8 @@ def serialize(
         warning_extra: Dict[str, str],
         event_id: Optional[str],
         detailed: bool = False,
+        organization: Organization = None,
+        request: HttpRequest = None,
```
Comment (k-fish) [inline src/sentry/api/endpoints/organization_events_trace.py:627]: Let's check the feature flag outside the serialize function and pass in the result of that flag check as something like `allow_orphan_errors` instead

**Review by k-fish — APPROVED** [review]
Aside from my suggestion to minimize passed params, looks good 👍

**Review by narsaynorath — APPROVED** [review]
+1 to Kevan's comment. Nice changes!

File: src/sentry/api/endpoints/organization_events_trace.py (hunk 1 above)
Comment (Abdkhan14) [reply src/sentry/api/endpoints/organization_events_trace.py]: Agree.  Addressed this.

**Pull request merged**